dotnet/packages/Microsoft.Bot.Builder.M365/Microsoft.Bot.Builder.M365.Tests/IntegrationTests/AzureContentSafetyModeratorTests.cs

```diff
@@ -0,0 +1,110 @@
+﻿using Microsoft.Bot.Builder.M365.AI.Moderator;
+using Microsoft.Bot.Builder.M365.AI.Planner;
+using Microsoft.Bot.Builder.M365.AI.Prompt;
+using Microsoft.Bot.Builder.M365.AI;
+using Microsoft.Extensions.Configuration;
+using Moq;
+using System.Reflection;
+using Xunit.Abstractions;
+using Microsoft.Bot.Builder.M365.AI.Action;
+using Microsoft.Bot.Schema;
+
+namespace Microsoft.Bot.Builder.M365.Tests.Integration
+{
+    public sealed class AzureContentSafetyModeratorTests
+    {
+        private readonly IConfigurationRoot _configuration;
+        private readonly RedirectOutput _output;
+
+        public AzureContentSafetyModeratorTests(ITestOutputHelper output)
+        {
+            _output = new RedirectOutput(output);
+
+            var currentAssemblyDirectory = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+
+            if (string.IsNullOrWhiteSpace(currentAssemblyDirectory))
+            {
+                throw new InvalidOperationException("Unable to determine current assembly directory.");
+            }
+
+            var directoryPath = Path.GetFullPath(Path.Combine(currentAssemblyDirectory, $"../../../IntegrationTests/"));
+            var settingsPath = Path.Combine(directoryPath, "testsettings.json");
+
+            _configuration = new ConfigurationBuilder()
+                .AddJsonFile(path: settingsPath, optional: false, reloadOnChange: true)
+                .AddEnvironmentVariables()
+                .AddUserSecrets<AzureOpenAICompletionTests>()
+                .Build();
+        }
+
+        [Theory]
+        [InlineData("I hate you", true)]
+        [InlineData("Turn on the light", false)]
+        [InlineData("我恨你", true)]
+        [InlineData("電気をつける", false)]
+        public async Task AzureContentSafetyModerator_ReviewPrompt(string input, bool flagged)
```

Comment:
These tests would fail if there is no working configuration (api key, endpoint) so we should make them manual tests by skipping them for now.

Reply:
Thanks for the notice. Skipped this test case.